src/app_service/service/activity_center/service.nim

```diff
@@ -115,8 +113,7 @@ QtObject:
     let callResult = backend.activityCenterNotifications(cursorVal, DEFAULT_LIMIT)
     let activityCenterNotificationsTuple = parseActivityCenterNotifications(callResult.result)
 
-    self.cursor = activityCenterNotificationsTuple[0];
-
+    # self.cursor = activityCenterNotificationsTuple[0];
```

Comment:
Why commented out?

Reply:
Temporary, my second issue is about cursor fixes. Turned it back for this PR